kornia/geometry/plane.py

```diff
@@ -0,0 +1,55 @@
+# kornia.geometry.plane module inspired by Eigen::geometry::Hyperplane
+# https://gitlab.com/libeigen/eigen/-/blob/master/Eigen/src/Geometry/Hyperplane.h
+
+from typing import Optional
+
+from kornia.core import Module, Parameter, Tensor, normalize
+from kornia.testing import KORNIA_CHECK_SHAPE
+
+__all__ = ["Hyperplane"]
+
+
+class Hyperplane(Module):
+    def __init__(self, n: Tensor, d: Tensor) -> None:
+        super().__init__()
+        KORNIA_CHECK_SHAPE(n, ["B", "*"])
+        KORNIA_CHECK_SHAPE(d, ["B", "*"])
```

Comment:
@ducha-aiki how could we improve KORNIA_CHECK_SHAPE so that it accepts ["B", ("2", "3")] ?